glide.lock

```diff
@@ -138,4 +138,6 @@ imports:
   version: 61b46af70dfed79c6d24530cd23b41440a7f22a5
   subpackages:
   - pkg/common
-testImports: []
+testImports:
+- name: github.com/kr/text
```

Comment:
I don't see this being used anywhere. Please remove

Reply:
done